### PR TITLE
added Node.js

### PR DESCRIPTION
**Pull request by moosingin3space**

### CODE REVIEW

File: caenbrew/packages/nodejs.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from ..packaging import package, AutotoolsPackage
```
Comment (arxanas) [inline caenbrew/packages/nodejs.py]: I think you're going to be yelled at for not putting these in alphabetical order?

You can `pip install -r requirements-dev.txt` then run `flake8 caenbrew` to catch flakey issues early.

Comment (moosingin3space) [inline caenbrew/packages/nodejs.py]: Just did.

**Comment by arxanas** [comment]
Have you successfully installed and run anything with `npm`? I tried installing `bower`, `tldr`, and `bitbucket-snippet`, but all of them either had an error during installation or runtime.

**Comment by moosingin3space** [comment]
That's what I'm currently testing -- whether or not I can build a large Node app.

**Comment by moosingin3space** [comment]
Regarding `tldr` - it appears to work for me.

**Comment by arxanas** [comment]
It seems to have hung; how long did it take you to install? Are you installing with `-g`?

```
$ npm install -g tldr
npm WARN deprecated graceful-fs@3.0.8: graceful-fs version 3 and before will fail on newer node releases. Please update to graceful-fs@^4.0.0 as soon as possible.
/home/wkhan/.local/bin/tldr -> /home/wkhan/.local/lib/node_modules/tldr/bin/tldr

> tldr@1.6.0 postinstall /home/wkhan/.local/lib/node_modules/tldr
> node ./bin/tldr --update

/home/wkhan/.local/lib/node_modules/tldr/node_modules/glob/sync.js:128
    this._processGlobSta


SyntaxError: Unexpected token ILLEGAL
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:387:25)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at Object.<anonymous> (/home/wkhan/.local/lib/node_modules/tldr/node_modules/glob/glob.js:51:16)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
postinstall               ▐ ╢████████████████████████████████████████████████████████████████████████████░░░░░░░░░░░░╟
```

**Comment by moosingin3space** [comment]
It hangs when I try to install `atom`. I'll inspect the `nvm` source to see if I can find a better way to install Node.
